app/controllers/requests.js

```diff
@@ -0,0 +1,33 @@
+(function(exports) {
+  'use strict';
+
+  exports.RequestController = function(promoListModel, promoListView, RequestClass) {
+    var _promoListModel = promoListModel;
+    var _promoListView = promoListView;
```

Comment:
I don't feel like these two lines add anything - you could just use the injected variables promoListModel and promoListView.